src/database/migrations/20241122100559-create-table-rollouts.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		await queryInterface.createTable('rollouts', {
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: Sequelize.INTEGER,
+			},
+			resource_type: {
+				allowNull: false,
+				unique: true,
+				type: Sequelize.STRING,
+			},
+			resource_id: {
+				allowNull: false,
+				type: Sequelize.INTEGER,
+			},
+			status: {
+				allowNull: false,
+				type: Sequelize.ENUM('PENDING', 'ROLLED_OUT', 'INACTIVE'),
+				defaultValue: 'PENDING',
+			},
+			rollout_date: {
```

Comment:
@adithyadinesh0412  make this as a nullable field

Reply:
Default it should be pending right? @priyanka-TL . Can we have Null in status?

Reply:
rollout_date